tests/test_surface.py

```diff
@@ -329,6 +324,53 @@ def test_explicit_mesh(example_model_and_crs):
     assert_array_almost_equal(persistent_mesh.full_array_ref(), mesh.full_array_ref())
 
 
+@pytest.mark.parametrize('flavour,infile,filetype', [('explicit', 'Surface_roxartext.txt', 'roxar'),
```

Comment:
We should check that there are no IP issues with regard to including test files in these formats in the public repository.